### PR TITLE
turnstiles block atmos

### PR DESCRIPTION
**Pull request by Bibby0110**
Turnstiles now do not allow atmos

🆑
tweak: Turnstiles now do not allow atmos
/🆑

### CODE REVIEW

**Comment by Xantam-13** [comment]
@TheGamerdk Fix the content of your PR please... even if you just remove the content barring the changelog...

**Review by ToasterBiome — DISMISSED** [review]
changelog

**Comment by JamieD1** [comment]
Fixed Changelog

File: code/game/machinery/doors/turnstile.dm [hunk 1]
```diff
@@ -15,6 +15,10 @@
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 	layer = OPEN_DOOR_LAYER
 	climbable = FALSE
+	CanAtmosPass = ATMOS_PASS_NO
+
+
```
Comment (JamieD1) [inline code/game/machinery/doors/turnstile.dm]:
```suggestion
```

Comment (JamieD1) [inline code/game/machinery/doors/turnstile.dm]:
```suggestion
	CanAtmosPass = ATMOS_PASS_NO
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**